app/views/govuk_publishing_components/components/_image_card.html.erb

```diff
@@ -0,0 +1,56 @@
+<%
+  href ||= false
+  image_src ||= false
+  image_alt ||= ""
+  heading_text ||= false
+  metadata ||= false
+  description ||= false
+  large ||= false
+  heading_level ||= 2
+  heading_tag = "h#{heading_level}" if [1, 2, 3, 4, 5, 6].include? heading_level
+  extra_links ||= []
+  extra_links_no_indent ||= false
```

Comment:
Out of interest, where is `extra_links_no_indent` used/not used?

Reply:
It's used [here](https://govuk-publishing-compon-pr-322.herokuapp.com/component-guide/image_card/with_extra_links/preview) and not used [here](https://govuk-publishing-compon-pr-322.herokuapp.com/component-guide/image_card/extra_links_without_indent/preview).

Reply:
Sorry, I meant where on the site will we use https://govuk-publishing-compon-pr-322.herokuapp.com/component-guide/image_card/with_extra_links/preview? I don't see it on the current org page.

Reply:
Okido. As an alternative, would it work to use a block to populate the description, and then use a list component inside it?

```erb
<%= render '.../image_card', href: "/foo" do %>
  <%= render '.../list', items: [{ text: "About the Number 10 guest historian series", href: "/bar"}] %>
<% end %>
```

Reply:
That could work, although I'm not sure we have list components that visually match what I need. Also do we have a mechanism for documenting that in the yml file?